source/pages/Home/Home.tsx

```diff
@@ -125,7 +125,7 @@ export const Home = () => {
                   ? null
                   : moreThanMillion
                   ? formatMillionNumber(Number(fiatPriceValue))
```

Comment:
Here when u try to convert the Fiat Price to Number it get a NaN error and it is displayed at the Home screen. It happens because u try to convert a string like "$ 2852340  USD" into a Number and it's not possible. To prevent and handle it u can do a match case at this string like:

 `Number(fiatPriceValue.match(/[\d\.]+/g)[0])` 
 
 This will only get the Numbers / decimals places into it and the problem will be solved.

Reply:
"Got it. As we mentioned during our daily meeting today, this issue already existed and wasn't related to the function I created. However, your point makes sense, and I will fix it. Well done mate"

Reply:
FIxed